<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <widget class="QPushButton" name="settingsPushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="levelCreatorPushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Launch level creator</string>
   </property>
  </widget>
  <widget class="QPushButton" name="levelSelectorPushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Select level</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
